docker/netbeans-runner/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:18.04
```

Comment:
is this file meant to replace the other file `Dockerfile.netbeans-11.3` ?

Reply:
nope - i wasn't meant to commit this :\